### ISSUE TITLE
FEEDBACK: Geosearch and Map Coordinates

### ISSUE THREAD

**Issue by phil-wilson** [tedrick/Survey123-Testing#136]
While we have had geosearch in the map control of the Survey123 web app for quite some time, we did not have it in the Survey123 field app. The geosearch control in the field app is only available when the map is shown at full screen. The map preview in your survey will not provide an option to run a geosearch, but if you tap in the map to set its location manually, it will. 

The geosearch input accepts coordinates in different formats (Degrees Minutes Seconds, Degrees and Decimal Minutes, USNG and MGRS coordinates....). End users can also type a placename or address and a list of candidates sorted by distance from the device's location will be presented. 

By default, geosearch uses the locators that have been configured in the ArcGIS Organization.   As is commonplace through other geosearch inputs in ArcGIS (Web AppBuilder, Web Map Viewer etc), the end user can also explicitly define the locator to be used in the event that multiple locators have been configured in the ArcGIS organization.

Documentation can be found here: http://doc.arcgis.com/en/survey123/desktop/create-surveys/geopoints.htm#ESRI_SECTION1_48C66C92612A414F97F5D843DE8C909C

Example survey: [Geosearch.xlsx](https://github.com/tedrick/Survey123-Testing/files/2326071/Geosearch.xlsx)

**Comment by phil-wilson** [comment]
We want your feedback on geosearch and map coordinates, try the following:

- Design and publish a survey from Connect (see above example for ideas)
- Modify the map settings via Connect map settings page to set different coordinate formats
- Include different question types, calculations, defaults, constraints etc in your survey as well as a geopoint question
- Enable the Inbox to test your survey and when editing existing surveys
- Open your survey in the field app and try searchiing for names, places, coordinates in the search bar.
- Try the different options under the world option button and also try editing the existing coordinates or manually entering coordinate using the new controls.
- Test collecting new surveys, saving to drafts, outbox, sent folders and then also test editing via the inbox, drafts and sent.
- View your submitted surveys on the Survey123 website and AGO.

**Comment by JohnHasthorpe** [comment]
- Gee
- Paul
- Daniel
- Neale
- Ryan

**Comment by paul-gpsit** [comment]
I created a new survey from the "Calculate Location from GeoPoint" sample and downloaded in on my iPhone 6. I started to collect, and tapped the map to search for a location using the address search. I typed in an NZ address, then tapped it in the list. The actual map flicked to that location for a second then went straight back to my current GPS location. I tried other addresses and same thing - it kept flashing up the location then immediately flicking back to current gps location.
Finally I saved the location and went back a screen. The map graphic did correctly update with the location of the address.
But from then on it worked fine, i.e. every address search would update the map to that location and stay there, and I can't seem to replicate the original issue. Tried restart of app, and also tried creating a brand new survey, but it now seems to work fine.
Not sure what's going on but noting it here in case....
